src/sourmash/minhash.py

```diff
@@ -295,6 +295,23 @@ def add_sequence(self, sequence, force=False):
         self._methodcall(lib.kmerminhash_add_sequence, to_bytes(sequence),
                          force)
 
+    def seq_to_hashes(self, sequence, force=False, is_protein = False):
+        "Convert sequence to hashes without adding to the sketch."
+
+        if is_protein and self.moltype not in ["protein", "dayhoff"]:
```

Comment:
I think `hp` also goes here?

Reply:
also, you might want to check for mh is protein / `is_protein` is False.

Reply:
cherry-picked 32330b7d1f672c89a5c4a23a94aeef203a554896